spec/spec.md

```diff
@@ -565,8 +565,7 @@ The issuer generates a DIDComm v2 Out-Of-Band(OOB) invitation message with `goal
 
 #### Step 2 : Issue Credential - Propose Credential
 
-The wallet (user agent) initiates Issuer interaction by sending [Issue Credential - Propose Credential](https://github.com/decentralized-identity/waci-presentation-exchange/blob/main/issue_credential/README.md#propose-credential) message with `pthid` same as `id` from OOB message. This provides context to the Issuer by mapping the request to the original session.
-
+The wallet (user agent) initiates Issuer interaction by sending [Issue Credential - Propose Credential](https://github.com/decentralized-identity/waci-presentation-exchange/blob/main/issue_credential/README.md#propose-credential) message.
```

Comment:
I find this language a little confusing... I would prefer a bit more verbosity here... for example, I think the client (user agent / wallet) is actually proposing credential issuance, or "requesting a credential be issued to them).